README.md

```diff
@@ -56,6 +56,9 @@ jobs:
         with:
           notion-token: ${{ secrets.NOTION_TOKEN }}
           notion-db: ${{ secrets.NOTION_DATABASE }}
+
+          # This token is optional, it defaults to the built-in GITHUB_TOKEN.
+          # You can still use any PAT, provided that it has access to your issues.
           github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Maybe we could take this out since in the vast majority of cases, it won't be set. In this PR or as a separate issue, we could add a table to the README documenting the required and optional fields, but I think leaving this out is good for now